docs/rfc/000-Payload-v37.md

```diff
@@ -24,8 +24,11 @@ type PayloadAlpha37 = [
   // we can save more space
   authorNetwork: UTF8String | SocialNetworkEnum,
   authorID: UTF8String,
-  authorPublicKey: ArrayBuffer,
-  // authorPublicKeyCurve: ???,
+  //authorPublicKey: ArrayBuffer,
```

Comment:
Please leave this part. authorPublicKey is not part of Ephemeral encrytion process. It's persona discover

Reply:
And without `authorPublicKeyCurve` Mask cannot discover the persona correctly (don't know which curve on)

Reply:
With `SupportedCurves` we can actually append a single byte next to the `authorPublicKey` indicating the curve. What do you think?

Reply:
Yes, a CurveEnum is good